src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs

```diff
@@ -2494,6 +2494,8 @@ internal static string GetConsoleWindowTitle()
             return consoleTitle.ToString();
         }
 
+        private static bool dontSetConsoleWindowTitle;
```

Comment:
Our style:
```suggestion
        private static bool _dontSetConsoleWindowTitle;
```

Reply:
looks like the convention in the codebase is not an underscore, but pascal casing instead of camel

Reply:
@SteveL-MSFT Please look our guidelines we follow for years https://github.com/PowerShell/PowerShell/blob/master/docs/dev-process/coding-guidelines.md#naming-conventions

> Use _camelCase to name internal and private fields and use readonly where possible. Prefix instance fields with _, static fields with s_ and thread static fields with t_. When used on static fields, readonly should come after static (i.e. static readonly not readonly static).

This says us to use `s_dontSetConsoleWindowTitle`